packages/firebase_in_app_messaging/pubspec.yaml

```diff
@@ -25,7 +26,7 @@ flutter:
   plugin:
     platforms:
       android:
-        package: com.example.firebase_in_app_messaging
+        package: io.flutter.plugins.firebase.inappmessaging
```

Comment:
Woops 😅 

Should we maybe mention this in the changelog? I'm guessing Android hasn't been working at all before this due to the wrong package name?

Reply:
Will do, it was strange that the Android side used the `com.example...` package.

Reply:
Done